main/lsp/test/lsp_preprocessor_test.cc

```diff
@@ -64,23 +67,6 @@ bool comesBeforeSymmetric(const TimeTravelingGlobalState &ttgs, u4 a, u4 b) {
     return ttgs.comesBefore(a, b) && !ttgs.comesBefore(b, a);
 }
 
-unique_ptr<LSPMessage> makeOpen(string_view path, u4 version, string_view source) {
```

Comment:
I realized there were near-duplicate methods in ProtocolTest.h and test/helpers/lsp.h, so I migrated everything over to methods defined in `test/helpers/lsp.h`.